data/net/minecraft/world/level/storage/loot/IntRange.mapping

```diff
@@ -35,3 +35,12 @@ CLASS net/minecraft/world/level/storage/loot/IntRange
 		METHOD apply (Lnet/minecraft/world/level/storage/loot/LootContext;I)I
 			ARG 1 lootContext
 			ARG 2 value
+	CLASS Serializer
+		METHOD deserialize (Lcom/google/gson/JsonElement;Ljava/lang/reflect/Type;Lcom/google/gson/JsonDeserializationContext;)Lnet/minecraft/world/level/storage/loot/IntRange;
+			ARG 1 elementJson
+			ARG 2 type
+			ARG 3 context
```

Comment:
Is this the rule for any method overriding another one?

Reply:
Yes (at least in most cases). Methods that override other methods within the Minecraft source code are auto-populated, so you only need to worry about cases where a method overrides one from an open-source library. In this case, this overrides `JsonSerializer#serialize` from Google Gson. As for the case for `ResourceLocation$Serializer#serialize`, this will need to changed by someone.